internal/appstore/appstore.go

```diff
@@ -197,7 +189,7 @@ func calculateBundleHash(filename string) (string, error) {
 }
 
 func createLocateRequest(url, username, accesskey string, queryString string) (*http.Request, error) {
-	req, err := requesth.New(http.MethodGet, fmt.Sprintf("%s%s&per_page=100", url, queryString), nil)
+	req, err := requesth.New(http.MethodGet, fmt.Sprintf("%s%s", url, queryString), nil)
```

Comment:
you could reduce the amount of items per page to 1

Reply:
ok!